study-builder/studies/brugada/patches/addKitConfigurationAndUpdateEnrolledEvent.conf

```diff
@@ -0,0 +1,30 @@
+"events": [
```

Comment:
we might want to add these events to study-events.conf since we are not adding the patch to patch.log

Reply:
It's already set up and ready to work the same way either from scratch or after patch application.